### ISSUE TITLE
Discovery gives error in the Fabric restserver in 2.0.0

### ISSUE THREAD

**Issue by sownak** [hyperledger-labs/blockchain-automation-framework#649]
**Describe the bug**
When executing the APIs via Fabric Restserver, there is an error in the logs as follows (This is applicable to Fabric 2.0.0
```
2020-04-30T12:10:38.259Z - error: [Channel.js]: Channel:allchannel received discovery error:failed constructing descriptor for chaincodes:<name:"_lifecycle" >
2020-04-30T12:10:38.259Z - error: [Channel.js]: Not able to get an endorsement plan for {"chaincodes":[{"name":"_lifecycle"}]}
```

**To Reproduce**
1. Deploy Fabric 2.0.0 network using BAF
2. Deploy supplychain chaincodes and restserver
3. Execute the /product or /container API
4. check the logs of the restserver

**Expected behavior**

**Screenshots**
This is the error message on RESTserver when discovery is enabled
![image](https://user-images.githubusercontent.com/20537268/80709207-d2adbf00-8ae4-11ea-993f-f6c49264a69d.png)

**Additional context**
This maybe fixed when chaincode lifecycle is enabled
https://hyperledger-fabric.readthedocs.io/en/release-2.0/enable_cc_lifecycle.html

**Comment by lakshyakumar** [comment]
![image](https://user-images.githubusercontent.com/56861637/98327093-c6b53380-2018-11eb-85fa-15547f9c6bd7.png)
After restart the resterver logs were not showing discovery error as seen in the screenshot above for fabric 2.2.0 with a chaincode lifecycle.

**Comment by lakshyakumar** [comment]
RCA: The discovery required the support of chaincode lifecycle in fabric version 2.0.0, got resolved as the chaincode lifecycle support is implemented in fabric version 2.2.0
